packages/server/src/migrations/schema/v57.ts

```diff
@@ -0,0 +1,182 @@
+  'DeviceRequest',
+  'DeviceUseStatement',
+  'DiagnosticReport',
+  'DocumentManifest',
+  'DocumentReference',
+  'DomainConfiguration',
+  'EffectEvidenceSynthesis',
+  'Encounter',
+  'Endpoint',
+  'EnrollmentRequest',
+  'EnrollmentResponse',
+  'EpisodeOfCare',
+  'EventDefinition',
+  'Evidence',
+  'EvidenceVariable',
+  'ExampleScenario',
+  'ExplanationOfBenefit',
+  'FamilyMemberHistory',
+  'Flag',
+  'Goal',
+  'GraphDefinition',
+  'Group',
+  'GuidanceResponse',
+  'HealthcareService',
+  'ImagingStudy',
+  'Immunization',
+  'ImmunizationEvaluation',
+  'ImmunizationRecommendation',
+  'ImplementationGuide',
+  'InsurancePlan',
+  'Invoice',
+  'JsonWebKey',
+  'Library',
+  'Linkage',
+  'List',
+  'Location',
+  'Login',
+  'Measure',
+  'MeasureReport',
+  'Media',
+  'Medication',
+  'MedicationAdministration',
+  'MedicationDispense',
+  'MedicationKnowledge',
+  'MedicationRequest',
+  'MedicationStatement',
+  'MedicinalProduct',
+  'MedicinalProductAuthorization',
+  'MedicinalProductContraindication',
+  'MedicinalProductIndication',
+  'MedicinalProductIngredient',
+  'MedicinalProductInteraction',
+  'MedicinalProductManufactured',
+  'MedicinalProductPackaged',
+  'MedicinalProductPharmaceutical',
+  'MedicinalProductUndesirableEffect',
+  'MessageDefinition',
+  'MessageHeader',
+  'MolecularSequence',
+  'NamingSystem',
+  'NutritionOrder',
+  'Observation',
+  'ObservationDefinition',
+  'OperationDefinition',
+  'OperationOutcome',
+  'Organization',
+  'OrganizationAffiliation',
+  'Parameters',
+  'PasswordChangeRequest',
+  'Patient',
+  'PaymentNotice',
+  'PaymentReconciliation',
+  'Person',
+  'PlanDefinition',
+  'Practitioner',
+  'PractitionerRole',
+  'Procedure',
+  'Project',
+  'ProjectMembership',
+  'Provenance',
+  'Questionnaire',
+  'QuestionnaireResponse',
+  'RelatedPerson',
+  'RequestGroup',
+  'ResearchDefinition',
+  'ResearchElementDefinition',
+  'ResearchStudy',
+  'ResearchSubject',
+  'RiskAssessment',
+  'RiskEvidenceSynthesis',
+  'Schedule',
+  'SearchParameter',
+  'ServiceRequest',
+  'Slot',
+  'SmartAppLaunch',
+  'Specimen',
+  'SpecimenDefinition',
+  'StructureDefinition',
+  'StructureMap',
+  'Subscription',
+  'SubscriptionStatus',
+  'Substance',
+  'SubstanceNucleicAcid',
+  'SubstancePolymer',
+  'SubstanceProtein',
+  'SubstanceReferenceInformation',
+  'SubstanceSourceMaterial',
+  'SubstanceSpecification',
+  'SupplyDelivery',
+  'SupplyRequest',
+  'Task',
+  'TerminologyCapabilities',
+  'TestReport',
+  'TestScript',
+  'User',
+  'UserConfiguration',
+  'ValueSet',
+  'VerificationResult',
+  'VisionPrescription',
+];
+
+export async function run(db: PoolClient): Promise<void> {
+  for (const resourceType of resourceTypes) {
+    const start = Date.now();
+    await db.query(
+      `CREATE INDEX CONCURRENTLY IF NOT EXISTS "${resourceType}_text_tsv_idx" on "${resourceType}_Token" USING gin (to_tsvector('simple', value)) WHERE system = 'text'`
```

Comment:
I think this index name should be `${resourceType}_Token_text_idx_tsv` -

1. Add `_Token`
2. Align with our other indexes, where if the `_tsv` is implicit, we add it after `_idx`